jimin/prompt.py

```
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
```

Ruff (0.13.1)

Shebang is present but file is not executable

(EXE001)